protocols/raft/src/main/java/io/atomix/protocols/raft/roles/PassiveRole.java

```diff
@@ -842,16 +857,15 @@ public void incrementOffset() {
     /**
      * Commits the snapshot to disk.
      */
-    public void commit() {
-      snapshot.persist().complete();
+    public void commit() throws Exception {
```

Comment:
I think it's not necessary to declare exception here.

Reply:
Ah, yes, left over from when I had `complete` also throw an Exception. :see_no_evil: